.changes/windows-modify-cmd-string-behaviour.md

```diff
@@ -0,0 +1,5 @@
+---
+"cli.rs": patch
+---
+
+Change `beforeDevCommand` and `beforeBuildCommand` to run with `CMD /S /C {command}` instead of `CMD /C {command}` on Windows.
```

Comment:
I think the problem here is more important than the fix itself so the changelog should reflect that, somthing like this maybe.

```suggestion
On Windows, Fix `beforeDevCommand` and `beforeBuildCommand` not executing the expected command if it contains quotes. This is done by executing them with `CMD /S /C {command}` instead of `CMD /C {command}` on Windows.
```